boards/arm/nrf52_pca20020/board.c

```diff
@@ -71,8 +72,8 @@ static const struct pwr_ctrl_cfg ccs_vdd_pwr_ctrl_cfg = {
 	.pin = CCS_VDD_PWR_CTRL_GPIO_PIN,
 };
 
-DEVICE_INIT(ccs_vdd_pwr_ctrl_init, "", pwr_ctrl_init, NULL,
+DEVICE_AND_API_INIT(ccs_vdd_pwr_ctrl_init, "", pwr_ctrl_init, NULL,
 	    &ccs_vdd_pwr_ctrl_cfg, POST_KERNEL,
-	    CONFIG_BOARD_CCS_VDD_PWR_CTRL_INIT_PRIORITY);
+	    CONFIG_BOARD_CCS_VDD_PWR_CTRL_INIT_PRIORITY, NULL);
```

Comment:
Passing a NULL for api isn't doing anything but adding bytes to the source file.  This is just duplicating the behavior of DEVICE_INIT().

Reply:
@andyross  yes, plan is to deprecate DEVICE_INIT() so this change(using DEVICE_AND_API_INIT) is necessary, without this  error will returned as still using  DEVICE_INIT()  although deprecated.

Pls check: https://github.com/zephyrproject-rtos/zephyr/pull/11984#issuecomment-445937243